src/classes/TestDataClass.cls

```diff
@@ -49,13 +49,23 @@ public with sharing class TestDataClass {
         return cdl;
     }
 
-    public static User insertUser() {
+    public static List<User> insertUser() {
         Profile p = [SELECT Id FROM Profile WHERE Name = 'OCInvoice User'];
-        User u = new User(Alias = 'newUser', Email = 'newuser@testorg.com',
+        List<User> u = new List<User>{new User (Alias = 'newUser', Email = 'newuser@testorg.com',
```

Comment:
Przekombinowane - tutaj nie potrzebujesz listy user'a do insertu. Wystarczy zainsertować zwykłego user'a i zwrócić listę budowaną w ten sposób. Puszczam to dalej, ale miej w pamięci toto :)